File: exs/mundo_1/golang/013.go
/*
Desafio 013

Problema: Faça um algoritmo que leia o salário de um funcionário
          e mostre seu novo salário, com 15% de aumento.

Resolução do problema:
*/

package main

import (
	"fmt"
)

var salario float64

func main(){
	fmt.Print("Digite o salário do funcionário: ")
	fmt.Scanln(&salario)

	aumento := 0.15 //15% de aumento
	salarioComAumento := salario + (salario * aumento)
	fmt.Println("O salário de", salario ,"com 15% de aumento fica",salarioComAumento)
}

File: exs/mundo_1/c#/013.cs
/*
Desafio 013

Problema: Faça um algoritmo que leia o salário de um funcionário 
          e mostre seu novo salário, com 15% de aumento

Resolução do problema:
*/

using System;

static class io {
    public static string input(string text) {
        Console.Write(text);
        return Console.ReadLine();
    }
    
    public static void print(object data, string end="\n") {
        string text = data.ToString();

        Console.Write($"{text}{end}");
    }
}

class Program {
    public static void Main() {
        double salario = double.Parse(
          io.input("Informe seu sálario: R$")
        );

        double salarioFinal = Math.Truncate((salario * 1.15 * 100)) / 100;

        io.print($"O salario final é {salarioFinal}");
    }
}